mis_builder/tests/test_render.py

```diff
@@ -209,6 +212,7 @@ def test_merge(self):
         self.assertEquals(style_props, {
             'color': '#00FF00',
             'dp': 1,
+            'hide_always': False,
         })
```

Comment:
why do you need to change this test?

Reply:
'hide_always' does not have a corresponding 'hide_always_inherit', and correspondingly will be returned here https://github.com/OCA/mis-builder/blob/11.0/mis_builder/models/mis_report_style.py#L157

Reply:
Perhaps I should add 'hide_always_inherit'?

Reply:
I was wondering too

Reply:
Hmm, now that I think about it, when you hide a higher level it should always hide lower levels. So I will create 'hide_always_inherit'.